service_identity/src/main/kotlin/org/thechance/service_identity/endpoints/userManagementRoutes.kt

```diff
@@ -1,48 +1,44 @@
 package org.thechance.service_identity.endpoints
 
 import io.ktor.server.application.*
+import io.ktor.server.request.*
 import io.ktor.server.response.*
 import io.ktor.server.routing.*
 import org.koin.ktor.ext.inject
 import org.thechance.service_identity.data.mappers.toDto
 import org.thechance.service_identity.domain.entity.MissingParameterException
-import org.thechance.service_identity.domain.usecases.user_management.UserManagementUseCase
-import org.thechance.service_identity.endpoints.validation.INVALID_REQUEST_PARAMETER
+import org.thechance.service_identity.domain.usecases.user_management.IUserManagementUseCase
+import org.thechance.service_identity.domain.usecases.util.INVALID_REQUEST_PARAMETER
 
 fun Route.userManagementRoutes() {
 
-    val userManagement: UserManagementUseCase by inject()
+    val userManagement: IUserManagementUseCase by inject()
 
     route("/dashboard/user") {
 
-        get("/{id}") {
-            val id = call.parameters["id"] ?: ""
-            userManagement.getUserById(id)
-        }
-
         get {
             val fullName = call.parameters["full_name"] ?: ""
             val username = call.parameters["username"] ?: ""
-            val users = userManagement.getUsersList(fullName, username)
+            val users = userManagement.getUsers(fullName, username)
```

Comment:
do mapping here and send http code

Reply:
I haven't got it, can you explain please?

Reply:
val users = userManagement.getUsers(fullName, username).toDto()
            call.respond(HttpStatusCode.OK, users)

Reply:
got it